build-kong.sh

```diff
@@ -44,9 +44,19 @@ pushd /kong
   mkdir -p /tmp/build/etc/kong
   cp kong.conf.default /tmp/build/usr/local/lib/luarocks/rock*/kong/$ROCKSPEC_VERSION/
   cp kong.conf.default /tmp/build/etc/kong/kong.conf.default
+
+  # clone dp-spec
+  DP_SPEC = /tmp/kong-dp-spec
+  git clone https://github.com/Kong/kong-dp-spec ${DP_SPEC} --recursive
+  git -C ${DP_SPEC} checkout ${KONG_DP_SPEC_VERSION:-f9432f8}
   # /usr/local/kong/include is usually created by other C libraries, like openssl
   # call mkdir here to make sure it's created
   mkdir -p /tmp/build/usr/local/kong/include
+  # wrpc related protos
+  cp -r ${DP_SPEC}/spec/proto/* /tmp/build/usr/local/kong/include/
+  mkdir /tmp/build/usr/local/kong/include/wrpc
```

Comment:
use `mkdir -p`?

Reply:
We are sure the parent directory exists, and wrpc does not